src/app/blog/components/post-section.tsx

```
@@ -0,0 +1,115 @@
+import {
+  BlogPostFile,
+  BlogSection,
+  BlogSectionType,
+  PostMarketSlug,
+  PostSummaryText,
+  PostTable,
+  PostTableRow,
+  PostValue,
+} from '@/types/blog'
+
+interface ListItem {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ListItem'. Consider to delete it or rename it.

'ListItem' is defined here:


(lint/suspicious/noRedeclare)